From 30dd09eae3ef6e2f1d005323631b505ea5dff21b Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 8 Sep 2014 13:07:14 -0400
Subject: [tools] perf: Remove unlikely define from perf.h

Message-id: <1410181752-25631-190-git-send-email-jolsa@redhat.com>
Patchwork-id: 91852
O-Subject: [PATCH RHEL7.1 BZ1134356 189/307] perf tools: Remove unlikely define from perf.h
Bugzilla: 1134356
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1134356
https://bugzilla.redhat.com/show_bug.cgi?id=1134356

upstream
========
commit 8be26c539f0d4c77a798b20515ad6404d72de74d
Author: Jiri Olsa <jolsa@kernel.org>
Date: Mon May 5 12:32:05 2014 +0200

description
===========
It's defined in tools/include/linux/compiler.h header.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/perf.h b/tools/perf/perf.h
index 3bea2e6..a547f38 100644
--- a/tools/perf/perf.h
+++ b/tools/perf/perf.h
@@ -203,7 +203,6 @@ static inline unsigned long long rdclock(void)
  */
 #define asmlinkage
 
-#define unlikely(x) __builtin_expect(!!(x), 0)
 #define min(x, y) ({    \
  typeof(x) _min1 = (x);   \
  typeof(y) _min2 = (y);   \
-- 
1.7.1